apps/labs/components/Post/PostMetaSection/PostMetaSection.tsx

```diff
@@ -22,7 +22,7 @@ export const PostMetaSection: FC<PostMetaSectionProps> = ({
         Published {published}
       </p>
       <div className="mb-[5rem]">
-        <PostAuthor {...author} />
+        <PostAuthor authors={author} />
```

Comment:
Plural on everything here, to match the other changes.

```suggestion
        <PostAuthors authors={authors} />
```